### DISCUSSION TITLE
Loki graduation to stable

### DISCUSSION THREAD

**Discussion by luisloros** [vectordotdev/vector#17201] [Q&A] [answered]
Hi Vector Team, with Loki getting some traction among the community, I was wondering how is the graduation to stable for the Loki sink in Vector is going?
Or in another way of putting the question, what are the issues that need fixing, to graduate Loki sink from beta to stable?

**Comment by jszwedko** [comment, marked as answer]
Hey! We plan to do a review of component statuses soon. I expect `loki` will be promoted to stable as part of that.
